### PR TITLE
Production

### PR DESCRIPTION
**Pull request by TerryRPatterson**
Fixes Deployment issuses

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_BACKEND="https://digicaching.herokuapp.com"
```
Comment (BGamber) [inline .env.local]: Just checking, does this file need to be included in the commit?

Comment (TerryRPatterson) [reply .env.local]: It needs to be included in the heroku server I am not sure how to put it on the dyno otherwise

Comment (BGamber) [reply .env.local]: Roger. 👍

**Review by BGamber — APPROVED** [review]
Looks good! 👍

**Comment by kbooth1000** [comment]
Well done Terry!

**Pull request merged**